clis/teliod/src/comms.rs

```diff
@@ -58,6 +58,7 @@ impl DaemonSocket {
         let _ = fs::remove_file(ipc_socket_path);
         let socket = ListenerOptions::new()
             .name(ipc_socket_path.to_fs_name::<FilesystemUdSocket>()?)
+            .reclaim_name(true)
```

Comment:
Is'nt the reclamation enabled by default?

Reply:
It is - I'm not really sure why I've added it and will remove it in a sec 😕